[ Typecasting]  Added the function text to agtype

Zahid07 opened the pull request:
Added the function to cast text to agtype
Corrected the catalog test case

Useage:
`SELECT * FROM text_to_agtype('test');`

```
Output:
 text_to_agtype
----------------
 "test"
(1 row)
```

Zahid07 committed:
Added the casting text to agtype

Added the function to cast text to agtype
Corrected the catalog test case

JoshInnis commented on age--1.1.0.sql:
```diff
@@ -3198,6 +3198,14 @@ AS 'MODULE_PATHNAME';
+RETURNS NULL ON NULL INPUT
+PARALLEL SAFE
+AS 'MODULE_PATHNAME';
+
```
You now need to add the CREATE CAST command which allows text to be cast like so `SELECT '"This is a string"'::agtype::text;` Also you need to add regression tests

Zahid07 replied:
Okay, I'll add these

Zahid07 replied:
Added the changes

JoshInnis committed:
Add Intersect and Except Set Operators

JoshInnis committed:
Merge pull request #117 from AGEDB-INC/set_operators

Add Intersect and Except Set Operators

Zahid07 committed:
Added CAST for text to agtype

Added testcases for text to agtype
Changed the cypher_vle testcases to use the new CAST

Zahid07 committed:
Merge branch 'AGEDB-INC:PG14' into typecasting

JoshInnis commented on regress/sql/expr.sql:
```diff
@@ -2461,6 +2461,17 @@ SELECT * from cypher('list', $$MATCH (u) RETURN labels(u), u$$) as (Labels agtyp
+--
+
+SELECT text_to_agtype('test');
+SELECT '"This is a string"'::agtype::text;
```
This is testing agtype_to_text

Zahid07 replied:
Please review

Zahid07 committed:
Changed testcase
